sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.aest.controller.assessApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -98,6 +106,90 @@ describe("ApplicationOfferingChangeRequestAESTController(e2e)-assessApplicationO
     ).toBe(applicationOfferingChangeRequest.requestedOffering.id);
   });
 
+  it("Should access the student appeal of the current assessment for the provided application offering change request id.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource, undefined, {
+      applicationStatus: ApplicationStatus.Completed,
+    });
+    const user = await userRepo.save(createFakeUser());
+    const studentAppeal = await createFakeStudentAppeal({ application });
+    const offering = await db.educationProgramOffering.save(
+      createFakeEducationProgramOffering({ auditUser: user }),
+    );
+    const studentAssessment = await createFakeStudentAssessment({
+      auditUser: user,
+      application,
+      offering,
+      studentAppeal,
+    });
+    application.currentAssessment = studentAssessment;
+    await db.application.save(application);
+    const applicationOfferingChangeRequest =
+      await saveFakeApplicationOfferingRequestChange(
+        db,
+        {
+          application,
+        },
+        {
+          initialValues: {
+            applicationOfferingChangeRequestStatus:
+              ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
+          },
+        },
+      );
+    const note = "Some dummy note.";
+    const payload = {
+      note,
+      applicationOfferingChangeRequestStatus:
+        ApplicationOfferingChangeRequestStatus.Approved,
+    };
+    const token = await getAESTToken(AESTGroups.BusinessAdministrators);
+    const endpoint = `/aest/application-offering-change-request/${applicationOfferingChangeRequest.id}`;
+    // Act/Assert
+    await request(app.getHttpServer())
+      .patch(endpoint)
+      .send(payload)
+      .auth(token, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK);
+    // Check if the application offering change request was assessed as expected.
+    const updatedApplicationOfferingChangeRequest =
+      await db.applicationOfferingChangeRequest.findOne({
+        select: {
+          id: true,
+          applicationOfferingChangeRequestStatus: true,
+          assessedNote: { description: true },
+          application: {
+            id: true,
+            currentAssessment: {
+              id: true,
+              triggerType: true,
+              offering: { id: true },
+              studentAppeal: { id: true },
+            },
+          },
+        },
+        relations: {
+          assessedNote: true,
+          application: {
+            currentAssessment: { offering: true, studentAppeal: true },
+          },
+        },
+        where: { id: applicationOfferingChangeRequest.id },
+      });
+    expect(
+      updatedApplicationOfferingChangeRequest.application.currentAssessment
+        .id !== studentAssessment.id,
+    ).toBeTruthy();
+    expect(
+      updatedApplicationOfferingChangeRequest.application.currentAssessment
+        .offering.id !== offering.id,
+    ).toBeTruthy();
```

Comment:
To keep these `expect` methods more aligned with the last one, can we use the `not.toBe` as below?
```ts
expect(
  updatedApplicationOfferingChangeRequest.application.currentAssessment
    .offering.id,
).not.toBe(offering.id);
```